examples/resources/elasticstack_kibana_action_connector/resource.tf

```diff
@@ -10,3 +10,22 @@ resource "elasticstack_kibana_action_connector" "example" {
   })
   connector_type_id = ".index"
 }
+
+resource "elasticstack_kibana_action_connector" "pagerduty-connector" {
+  name = "pagerduty"
+  connector_type_id = ".pagerduty"
+  config = jsonencode({
+    apiurl = "https://events.pagerduty.com/v2/enqueue"
```

Comment:
I think it should be `apiUrl`.

Reply:
Hi @dimuon, I have implemented this locally on my machine. Somehow both apiurl and apiUrl works.

Reply:
So do you want me to keep it as apiUrl in final commit?